core-services/prow/02_config/openshift/console/_pluginconfig.yaml

```diff
@@ -33,6 +33,12 @@ label:
       assign_on:
       - label: backport-risk-assessed
       label: cherry-pick-approved
+    - allowed_users:
+      - jhadvig
+      - rhamilto
+      - spadgett
+      - TheRealJon
+      label: pf6-approval-required
```

Comment:
I think there's another file where the label needs to be defined so it's added to the repo. Do we know if the label gets removed later if we remove it from the config file?

We could go with something more generic that we can use again in the future. `additional-approval-required`? `approved-fixes-only`? I'm open to suggestions :) There's the existing critical fixes only label, but I don't think we can lock that down to just this set of approvers.

Reply:
> I think there's another file where the label needs to be defined so it's added to the repo.

Looks like release/core-services/prow/02_config/_labels.yaml

Reply:
Yeah, the wording is always hard. I like the `approved-fixes-only` 👍 

Will update.

Thanks @spadgett